Description: Copyright & License Clarifications
 These changes are backported from upstream corrections/clarifications
 made in reponse from upstream author Ben Klang to enquiry by package
 maintainer. Verbatim from email dated 2011-04-11:
 .
 "I have updated the license file.  The Sun copyright was an error and
 should not have been there, so I removed it.  You are free to redistribute
 that file along with all other files in the package according to its
 license, GPLv2."
--- ldap2dns-0.4.2.orig/ldap2dns.xml
+++ ldap2dns-0.4.2/ldap2dns.xml
@@ -1,10 +1,25 @@
 <?xml version="1.0"?>
 <!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
 <!--
-	Copyright 2004 Sun Microsystems, Inc.  All rights reserved.
-	Use is subject to license terms.
+	Copyright 2006-2011 Alkaloid Networks, LLC
+        Licensed under the terms of the GPL version 2. See
+        http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 
-	ident	"@(#)server.xml	1.3	04/12/03 SMI"
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+	$Id$
 -->
 <service_bundle type='manifest' name='ALKAldap2dns:ldap2dns'>
 
--- ldap2dns-0.4.2.orig/ldap2dns.c
+++ ldap2dns-0.4.2/ldap2dns.c
@@ -4,6 +4,22 @@
  * Copyright 2005-2008 by Ben Klang <ben@alkaloid.net>
  * Copyright 2000-2005 by Jacob Rief <jacob.rief@tiscover.com>
  * License: GPL version 2. See http://www.fsf.org for details
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301, USA.
  */
 
 #include <lber.h>
